tests/linearizability/failpoints.go

```diff
@@ -101,11 +112,19 @@ func (f killFailpoint) Trigger(t *testing.T, ctx context.Context, clus *e2e.Etcd
 		}
 	}
 
-	err := member.Start(ctx)
-	if err != nil {
-		return err
+	// get endpoints excluding the killed member client URLs
+	endpoints := make([]string, 0, len(clus.EndpointsV3()))
+	for _, ed := range clus.EndpointsV3() {
+		if ed != member.EndpointsV3()[0] {
+			endpoints = append(endpoints, ed)
+		}
 	}
-	return nil
+
+	if f.enableAuth {
+		require.NoError(t, addTestUserAuth(ctx, endpoints))
```

Comment:
Don't like that auth setup is part of failpoint injection. Those are totally separate things. Please move auth setup to cluster setup.

Reply:
I think that enabling auth is orthogonal to failure injection.